collectors/0/kafka_consumer_jmx.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import re
+import sys
+
+from collectors.lib.jmx_monitor import JmxMonitor
+
+MONITORED_MBEANS = ["kafka.consumer", ""]
+
+
+class KafkaConsumerJmxMonitor(JmxMonitor):
+    USER = "flume"
+    PROCESS_NAME = "org.apache.flume.node.Application"
+    METRIC_PREFIX = "flume"
```

Comment:
ah these makes this collector non-generic. could you make this a KafkaJmxMonitor util class similar to JmxMonitor, and consolidate both consumer and producer, and maybe even broker in there? then rename collectors to flume_kafka and kafka_broker that both make use of the KafkaJmxMonitor util.